adal/src/androidTest/java/com/microsoft/aad/adal/MockBrokerAccountService.java

```diff
@@ -74,9 +74,20 @@ public synchronized Bundle getBrokerUsers() throws RemoteException {
         @Override
         public synchronized Bundle acquireTokenSilently(Map requestParameters) throws RemoteException {
             final Bundle bundle = new Bundle();
-            if(requestParameters.containsKey("isConnectionAvailable")) {
+            if (requestParameters.containsKey("isConnectionAvailable")) {
                 bundle.putInt(AccountManager.KEY_ERROR_CODE, AccountManager.ERROR_CODE_NETWORK_ERROR);
                 bundle.putString(AccountManager.KEY_ERROR_MESSAGE, ADALError.DEVICE_CONNECTION_IS_NOT_AVAILABLE.getDescription());
+            } else if (requestParameters.containsKey("throwRemoteException")) {
+                throw new RemoteException();
+            } else if (requestParameters.containsKey("throwOperationCanceledException")) {
+                bundle.putInt(AccountManager.KEY_ERROR_CODE, AccountManager.ERROR_CODE_CANCELED);
```

Comment:
please make the key strings constants in case of mis-spellings.

Reply:
Sorry, I did not understand your comment here. AccountManager.KEY_ERROR_CODE is a static final string in AccountManager.

Reply:
I meant strings like "throwOperationCanceledException" and "throwRemoteException".